src/textacy/extract/triples.py

```diff
@@ -209,6 +210,7 @@ def direct_quotations(doc: Doc) -> Iterable[DQTriple]:
 
     Args:
         doc
+        min_quote_length - minimum distance (in tokens) between potentially paired quotation marks.
```

Comment:
ah, seeing that you've listed this as a function arg but it only comes in as a hard-coded constant value. feeling more strongly that we should move this into an arg!

Reply:
yeah that makes sense